docs/faq.md

```diff
@@ -32,7 +32,22 @@ If Larq helps you in your work or research, it would be great if you can cite it
 }
 ```
 
-If your paper is publicly available, feel free to also add it to the list of [Papers using Larq](https://github.com/larq/docs/edit/master/docs/papers.md).
+If your paper is publicly available, feel free to also add it to the list of papers using Larq below.
+
+## What papers have used Larq?
+
+One of the focuses of Larq is to accelerate research on neural networks with extremely low precision weights and activations.
+Here is a list of papers that have used Larq:
+* Helwegen, Koen, et al. ["Latent weights do not exist: Rethinking binarized neural network optimization."](https://papers.nips.cc/paper/8971-latent-weights-do-not-exist-rethinking-binarized-neural-network-optimization.pdf) *Advances in Neural Information Processing Systems (NeurIPS)*. 2019.
```

Comment:
This doesn't really render well on the website: https://docs-ezf5lam29.now.sh/faq/#what-papers-have-used-larq

I think you might need to add some empty lines.

Reply:
Added a blank line in dc1739d.